### PR TITLE
mediaconch: fix libxslt dependency

### PR DESCRIPTION
**Pull request by retokromer**
Should solve
https://github.com/MediaArea/MediaConch/issues/119

### CODE REVIEW

File: Library/Formula/mediaconch.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Mediaconch < Formula
   url "https://mediaarea.net/download/binary/mediaconch/15.12/MediaConch_CLI_15.12hotfix1_GNU_FromSource.tar.bz2"
   version "15.12"
   sha256 "4899d43c097c552b2f970da6362407f002c61f93b11e8be3cf79b29c4733fd06"
-  revision 1
+  revision 2
```
Comment (sjackman) [inline Library/Formula/mediaconch.rb]: No need to bump the revision for a change of dependencies.

File: Library/Formula/mediaconch.rb [hunk 2]
```diff
@@ -16,6 +16,7 @@ class Mediaconch < Formula
   depends_on "pkg-config" => :build
   depends_on "jansson"
   depends_on "libevent"
+  depends_on "libxslt"
```
Comment (sjackman) [inline Library/Formula/mediaconch.rb]: Please change this line to
```ruby
depends_on "libxslt" unless OS.mac?`
```
and move it to the end of the list of dependencies.
See [CONTRIBUTING.md](https://github.com/Linuxbrew/linuxbrew/blob/master/CONTRIBUTING.md)

**Comment by sjackman** [comment]
Please squash to a single commit. Thanks for your contribution, Reto.

**Comment by retokromer** [comment]
Sorry, @sjackman, I could not see how to quick squash or rebase from the GitHub web interface. I opened #796.

**Comment by sjackman** [comment]
No, I don't believe squash and rebase are features of the GitHub web interface. Would be nice though.

**Comment by sjackman** [comment]
Superseded by #796.

**Pull request closed without merging**